corporate/lib/analytics.py

```diff
@@ -80,9 +78,12 @@ def get_plan_data_by_remote_server() -> Dict[int, RemoteActivityPlanData]:  # no
         current_data = remote_server_plan_data.get(server_id)
         if current_data is not None:
             current_revenue = remote_server_plan_data[server_id].annual_revenue
+            current_plans = remote_server_plan_data[server_id].current_plan_name
+            # There should only ever be one CustomerPlan for a remote server with
+            # a status that is less than the CustomerPlan.LIVE_STATUS_THRESHOLD.
             remote_server_plan_data[server_id] = RemoteActivityPlanData(
-                current_status="Multiple plans",
-                current_plan_name="See support view",
+                current_status="ERROR: MULTIPLE PLANS",
+                current_plan_name=f"{current_plans}, {plan.name}",
```

Comment:
This seemed better than the whole dataset not loading or something similar. Tried to make it something that would stand out.